tests/eas/acceptance.py

```diff
@@ -85,6 +88,52 @@ def log_result(data, log_fh):
     logging.info(result_str)
     log_fh.write(result_str)
 
+@wrapt.decorator
+def experiment_test(wrapped_test, instance, args, kwargs):
+    for experiment in instance.executor.experiments:
+        tasks = experiment['wload'].tasks.keys()
+        try:
+            wrapped_test(experiment, tasks, *args, **kwargs)
+        except AssertionError as e:
+            trace_relpath = os.path.join(experiment['out_dir'], "trace.dat")
+            logging.error("Check trace file: " + os.path.abspath(trace_relpath))
```

Comment:
It's printed next to the failure report, the output ends up looking like this:
```
...
04:49:03  INFO    :       Executor -    <res_dir>/rtapp::wake_migration/1/trace_stat.json
04:49:03  INFO    : --------------------------------------------------------------------------------
04:49:03  INFO    : 
04:49:03  INFO    : ################################################################################
04:49:03  INFO    :       Executor - Experiments execution completed
04:49:03  INFO    : ################################################################################
04:49:03  INFO    :       Executor - Results available in:
04:49:03  INFO    :       Executor -       /vagrant/results/20160927_164800
Wake Migration: BIG -> LITLLE: 1 ... 04:49:07  ERROR   : Check trace file: /vagrant/results/20160927_164800/rtapp::wake_migration/1/trace.dat
ok
Wake Migration: BIG -> LITLLE: 2 ... 04:49:11  ERROR   : Check trace file: /vagrant/results/20160927_164800/rtapp::wake_migration/1/trace.dat
ok
Test First CPU ... ok
Wake Migration: LITTLE -> BIG: 1 ... 04:49:17  ERROR   : Check trace file: /vagrant/results/20160927_164800/rtapp::wake_migration/1/trace.dat
ok
Wake Migration: LITTLE -> BIG: 2 ... 04:49:20  ERROR   : Check trace file: /vagrant/results/20160927_164800/rtapp::wake_migration/1/trace.dat
ok

----------------------------------------------------------------------
Ran 5 tests in 91.952s

OK
```

I've just realised that's different from the original output (it shouldn't say "OK" at the end) so I'll go back and take a look. This `raise` might not have the semantics I thought it did.